src/rgb/schemata/fungible.rs

```diff
@@ -94,36 +102,37 @@ impl Rgb1 {
         //let ts_schema = &schema.transitions[PRIM_ISSUE_TS];
 
         let mut meta = rgb::Metadata::from_inner(vec![
-            metadata::Field { id: metadata::Type(0), val: metadata::Value::Str(String::from(ticker)) },
-            metadata::Field { id: metadata::Type(1), val: metadata::Value::Str(String::from(name)) },
-            metadata::Field { id: metadata::Type(5), val: metadata::Value::U8(precision) },
-            metadata::Field { id: metadata::Type(7), val: metadata::Value::U8(network.into()) },
+            metadata::Field { id: metadata::Type(Self::TICKER_FIELD as u16), val: metadata::Value::Str(String::from(ticker)) },
```

Comment:
Probably we just need to change the type of consts to `u16

Reply:
the issue was that `HashMap<u16, T>` is not serializable and there's a comment that says that it should be better to only leave the `HashMap<usize, T>` serializable.

So since those constants later are used as keys in the map, they have to be `usize` for the map to be serializable. i was planning to discuss this with you and decide if we want to make other types of maps serializable, or go the other way and cast to usize before using them as keys.

Reply:
A comment? hm... Yeah, let's discuss that and think on it